pkg/services/rpcsrv/server.go

```diff
@@ -1733,7 +1726,7 @@ func (s *Server) getStorage(ps params.Params) (any, *neorpc.Error) {
 
 	item := s.chain.GetStorageItem(id, key)
 	if item == nil {
-		return "", nil
+		return "", neorpc.ErrUnknownStorageItem
```

Comment:
Ditto.

Reply:
1. IIRC, it's not the only function that has changed its behaviour, see the previous comment.

2. You have to directly specify reference to the C# revision where behaviour was changed in the commit body. The commit message may look like:
```
rpcsrv: return error on unknown storage item from `getstorage`

Follow the reference implementation, the behaviour was changed in
https://github.com/neo-project/neo-modules/commit/237ef7d057265723571efde74e54001f68ce969d.
```

3. This issue does not close #2248.

Reply:
Fixed